src/components/staking/PageValidator.vue

```diff
@@ -10,17 +10,6 @@
   >
     <template v-if="validator.operatorAddress" slot="managed-body">
       <div class="button-container">
-        <button
```

Comment:
why did you remove this?

Reply:
this button doesn't really make sense. the validator profile is not _always_ visited after the validator list. why should the button say "back to validators" if you visit a validator profile from your portfolio? i don't like this kind of exception — we don't do this for any other routes. 

we could add a proper back button (`-1`) — but i don't think `Pages` actually need a back button.

Reply:
it is unrelated to this PR. sorry about that.

Reply:
We decided on having this in the past. Maybe change the wording? This is app focused design where you can't go back to the validators in one click.

Reply:
Actually I had on my TODO list transforming this button functionality so it could be `Back to Validators/ Back to Portfolio` depending where you were visiting from.

But yes, I don't think deleting it altogether is the way to go either

Reply:
> This is app focused design where you can't go back to the validators in one click.

but i think you can do this now with our mobile menu

Reply:
true